sandbox/settings.py

```diff
@@ -105,6 +105,8 @@
 
 STATIC_URL = '/static/'
 
-TEMPLATE_DIRS = (os.path.join(BASE_DIR, 'accounts/templates'), )
+TEMPLATE_DIRS = (os.path.join(BASE_DIR, 'accounts/templates'),
+os.path.join(BASE_DIR, 'templates')
+)
```

Comment:
To jest zewnętrzny katalog, dodany poza aplikację, dlatego trzeba to dodać inaczej nie przeszukuje tego katalogu (nie ma aplikacji) to jest luźny katalog.